docker.go

```diff
@@ -410,6 +418,42 @@ func (p Plugin) Exec() error {
 		}
 	}
 
+	if p.TarPath != "" && p.Dryrun {
+		if len(p.Build.Tags) > 0 {
+			tag := p.Build.Tags[0]
+			fullImageName := fmt.Sprintf("%s:%s", p.Build.Repo, tag)
+
+			if !p.Build.BuildxLoad {
+				fmt.Printf("Warning: To save images to tar, 'load' must be enabled. Adding the '--load' flag automatically.\n")
+				p.Build.BuildxLoad = true
+			}
+
+			if !imageExists(fullImageName) {
+				fmt.Printf("Warning: Image %s not found in local daemon, cannot save to tar\n", fullImageName)
+			} else {
```

Comment:
Aren't these errors (instead of warnings)?

Reply:
this along with few more warning will be updated to errors